packages/discovery/src/discovery/runDiscovery.ts

```diff
@@ -76,14 +74,7 @@ export async function dryRunDiscovery(
   )
 
   if (diff.length > 0) {
-    const messages = diffToMessages(
-      projectConfig.name,
-      await findDependents(projectConfig.name, configReader),
-      diff,
-    )
-    for (const message of messages) {
-      console.log(message)
-    }
+    console.log(diff)
```

Comment:
Dear reviewer. This is the only real change in functionality. Because diffToMessages and findDependents were heavily dependent on external factors like discord message size and the configs being in backend I decided to drop nicer formatting for now. This was the easiest to do while preserving a logical separation of the two packages.